datastore/core/query.py

```diff
@@ -235,7 +235,7 @@ def __repr__(self):
         return "Order('%s%s')" % (self.op, self.field)
 
     def __eq__(self, other):
-        return self.field == other.field and self.op == other.op
+        return self.field == other.type and self.op == other.op
```

Comment:
Are you sure this is correct? I think the work `field` in `Filter` actually means `field` and that this function is comparing equality between filter types.

Also this should probably have a safety check like `if not isinstance(other, Filter): return NotImplemented` so that it won't crash when used as part of an incorrect comparison.

Reply:
Oh. Sorry. My bad.